satellite/console/consoleweb/consoleql/query.go

```diff
@@ -137,6 +138,8 @@ func rootQuery(service *console.Service, mailService *mailservice.Service, types
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
 					email, _ := p.Args[FieldEmail].(string)
 
+					email = strings.ToLower(email)
```

Comment:
Lets move that into the service and also use it across the board as normalized entries in the database.

Reply:
but in our CreateUser method in console.Service we "normalize" email making in lowercase

Reply:
and its the only place where we can set an email

Reply:
Take a look at the current database and you will see that there are entries in it with "non" lowercase emails.

Reply:
The email casing must be preserved for presentation.

Reply:
why @egonelbre ?

Reply:
Let's say my name is written as "AbCdE", I would expect it to be displayed that way. I.e. I expect to enter my email as I use it and see as I use it. There are of course issues with unicode with regards to case transformation, but, don't change the name or email of a person.

We can internally have a canonical representation, but it shouldn't be showed to the user.

Reply:
In this case we should disable email normalizing during user registration and login as well

Reply:
Just to clarify, logging in should still work with differently cased email.

Reply:
Can someone explain why we're changing the case here?

Reply:
@stefanbenten @egonelbre I'm going to rewrite `satellitedb/user GetUserByEmail` to be case insensitive.
Also I'll Add case insensitive check to projectMembers and reset password queries as well.

What do you think about this solution?

Reply:
Yes, making `GetUserByEmail` can be case-insensitive.

However, just to clarify:

1. We need two fields `email` and `normalized_email`.
2. When we query by email, we'll use `normalized_email`.
3. When we show to the user, we'll use `email`.

`normalized_email` would be in this case lower-cased. `email` whatever the user used to sign-up.

Reply:
Also `normalized_email` might remove dots, sub-addressing etc.

Reply:
@egonelbre We can make normalization by querying like `` select * from users where UPPER(email) = UPPER(input).
Or this is bad approach?

Reply:
That approach might be bad performance wise, since I think that would cause a full scan of the users table. If we want some more complex canonicalization, then it would need some more complex things.

But, it could work for the short term.